test/helpers/gas/gas.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from 'hardhat'
+import { Contract, Signer } from 'ethers'
+
+export class GasMeasurement {
```

Comment:
Can this be used to measure the gas usage of any function? I assume so, just want to double check.

Reply:
That's the intention, and I think it achieves that.